index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mulinde Derrick Portfolio</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+    <link
+      href="https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="logo-placeholder">
+          <p class="logo">M.D</p>
+        </div>
+        <nav class="nav-links">
+          <img class="menu-btn" src="/images/menu-btn.svg" alt="Menu Button" />
+        </nav>
+      </header>
+      <main>
+        <h1 class="title">I'm Derrick. Nice to see you!</h1>
+        <p class="about-me">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+      </main>
+      <footer>
+        <h4 class="social-title">LET'S CONNECT</h4>
+        <div class="social-icons">
+          <a href="#"><i class="bx bxl-twitter"></i></a>
+          <a href="#"><i class="bx bxl-linkedin"></i></a>
+          <a href="#"><i class="bx bxl-github"></i></a>
+        </div>
```

Comment:
- [ ] Please note that this is also part of the headline section. Kindly move it to the section.